app/services/datastream_importer.rb

```diff
@@ -16,16 +16,6 @@ def initialize(datastream_filename)
   def import!
     Xccdf::Benchmark.transaction do
       save_all_benchmark_info
-      import_remediations
     end
   end
-
-  private
-
-  def import_remediations
-    RemediationsAPI.new(
-      Account.find_by(account_number: ENV['JOBS_ACCOUNT_NUMBER']) ||
-        Account.new
```

Comment:
It looks like some of our deployments rely on this fallback. I've added it back here: https://github.com/RedHatInsights/compliance-backend/pull/848

Reply:
Oh, sorry, I haven't noticed it. I saw that the env var was configured and continued with the changes.

Reply:
I guess it needs to also be able to work when there are no accounts, like in a smoke test run.